packages/ui-extensions/docs/surfaces/point-of-sale/staticPages/pages/versions.doc.ts

```diff
@@ -17,7 +17,98 @@ const data: LandingTemplateSchema = {
       type: 'Generic',
       anchorLink: 'compatibility-policy',
       title: 'Compatibility Policy',
-      sectionContent: '',
+      sectionContent: `
+# POS UI Extensions Versioning
+
+This guide explains how versioning works for POS UI Extensions, including how to manage API versions, understand release schedules, and ensure merchants can access your extensions at the right time.
+
+## Understanding POS UI Extensions Versioning
+
+POS UI Extensions follow Shopify's quarterly API versioning system while also depending on the Shopify POS app release schedule. You need to understand both aspects to plan your development and release strategy effectively.
+
+### API Versioning
+- Shopify releases POS UI Extensions versions quarterly (January, April, July, October) following the [Shopify API versioning schedule](https://shopify.dev/docs/api/usage/versioning)
+- Versions use the format \`YYYY-MM\` (e.g., \`2025-04\`)
+- Shopify supports each version for 12 months from its release date
+- Quarterly releases introduce new features
+
+### POS App Release Cycle
+The Shopify POS app follows its own release schedule, which determines when merchants can access new extension features:
+- Shopify releases POS app updates approximately every two weeks (please note that there will be exceptions)
+- Merchants can access new API version features only when:
+  1. You update your extension to use the appropriate API version
+  2. Merchants install a compatible POS app version released on or after the API release date
+
+## Version Support Timeline
```

Comment:
Are you wanting to replace the existing [Compatibility Policy](https://shopify.dev/docs/api/pos-ui-extensions/2025-04/versions#compatibility-policy)?